src/routes/messages.js

```diff
@@ -11,12 +11,12 @@ const init = router => {
       return res.send({ errors: ['Enter message id'] });
     }
 
-    try {
-      return res.send(messageService.findOne(id));
-    } catch (e) {
-      res.statusCode = 404;
-      return res.send({ errors: [e.message] });
-    }
+    return messageService.findOne(id)
+      .then((message) => { res.send(message); })
```

Comment:
What's the difference between `.then((message) => { res.send(message); })` and `.then(res.send)`?